build.sbt

```diff
@@ -122,6 +122,7 @@ lazy val api = crossProject(JVMPlatform, JSPlatform)
     mimaSettings,
     name := "twirl-api",
     jsEnv := nodeJs,
+    testFrameworks := List(new TestFramework("utest.runner.Framework")),
```

Comment:
aren't we actually using ScalaTest as a test framework in this project?

Reply:
As I understood it, this trick you tell it to not lookup the other frameworks, see Sébastien comment here: https://github.com/scala-js/scala-js/issues/3673#issuecomment-497930605

On the other hand, this is only affecting local builds with GraalVM (works NOT on my machine syndrome). We will probably move to scala.js 1.1.0 soon (see #341). So maybe not worth it.

Reply:
right, so since we're using ScalaTest as our framework, we should list ScalaTest here (and not load the others)?

Reply:
yes, but what I did is wrong. Stupid copy-n-paste from the comment in scala.js issue. "utest.runner.Framework" doesn't exists.

This is the default value of the settings:

```scala
sbt:twirl> testFrameworks
[info] compiler / testFrameworks
[info] 	List(TestFramework(org.scalacheck.ScalaCheckFramework), TestFramework(org.specs2.runner.Specs2Framework, org.specs2.runner.SpecsFramework), TestFramework(org.specs.runner.SpecsFramework), TestFramework(org.scalatest.tools.Framework, org.scalatest.tools.ScalaTestFramework), TestFramework(com.novocode.junit.JUnitFramework))
```

We should fix it to `TestFramework(org.scalatest.tools.Framework, org.scalatest.tools.ScalaTestFramework)` and drop all the others.

I will fix it.